### DISCUSSION TITLE
Nemo File Manger "Open File/Image/etc..." dialog menu should be entirely replaced with the default file manager

### DISCUSSION THREAD

**Discussion by SpeeQz1** [linuxmint/discussions#698] [Ideas]
Currently when an app doesn't implement a custom file picker and uses the system's default one like for opening images it opens a dialogue of the type "Open Image". The problem with that dialog is that it is much more uncomfortable to navigate and quickly use compared to just the regular file manager of nemo.

![image](https://github.com/user-attachments/assets/4718c953-c93a-47d0-bb19-dc14807dc082)

What I am proposing is that the dialog just be the regular file manager but with the necessary buttons added near the bottom part of the window.

![image](https://github.com/user-attachments/assets/3783910c-4d5c-4512-a49d-8fd63e295027)

**Comment by Secret-chest** [comment]
I fully agree. However, that bad file chooser is hardcoded into GTK+; the only way to change it is to fork GTK.

**Comment by Jopp-gh** [comment]
I think this is taste and habit. The only thing I miss from the file chooser is a proper thumbnail rendering as it fails sometimes (showing a generic thumbnail) and maybe some bigger thumbnail on the right hand side for the selected image.

**Comment by timfeierabend** [comment]
This stood out to me too. One of my first impressions of Mint was how limiting the file chooser was versus the file browser, considering how similar the look and feel of both windows was. I find myself avoiding the file chooser altogether.

I wonder if a patched version of GTK+3 with an enhanced file chooser could be created for the Mint distros as a drop-in replacement for the upstream GTK+3 library?

**Reply by SpeeQz1** [reply]
I've had the same experience as when I am sharing files on Discord I just use the manager instead as it's less work.

**Comment by Qronikarz** [comment]
And once again:
related feature request: https://github.com/linuxmint/nemo/issues/3068
